tests/test_util.py

```diff
@@ -220,6 +218,17 @@ def test_setup_web3(monkeypatch):
     assert setup_web3(logger)
 
 
+def test_config_rpc(monkeypatch):
+    monkeypatch.setenv("NETWORK_URL", "https://rpc-mumbai.maticvigil.com/")
+
+    with pytest.raises(Exception) as e:
```

Comment:
you can use `pytest.raises(Exception, match="Mismatch of chain IDs between configuration and events RPC!")` to shorten the code here, instead of checking the args later